KeenClient.podspec

```diff
@@ -21,8 +21,8 @@ Pod::Spec.new do |spec|
   spec.source       = { :git => 'https://github.com/keenlabs/KeenClient-iOS.git', :tag => spec.version.to_s }
   spec.source_files = 'KeenClient/*.{h,m}','Library/Reachability/*.{h,m}'
   spec.public_header_files = 'KeenClient/*.h'
-  spec.frameworks   = 'CoreLocation'
   spec.requires_arc = true
+  spec.frameworks = 'SystemConfiguration', 'CoreLocation', 'CFNetwork'
```

Comment:
I did see that. I first wasn't too concerned since CFNetwork is a system-available shared dynamic library, meaning it won't need to be loaded into memory to execute since it will almost certainly already be in use by some other process. But that argument isn't really that important, because Foundation already requires it for things like NSURLSession. Just to prove that, here's what the SDK has linked to before this change (CFNetwork isn't dynamically linked here):

```
$ otool -L Examples/objc/cocoapods/DerivedData/KeenClientExampleObjCCocoaPods/Build/Products/Debug-iphonesimulator/KeenClient/KeenClient.framework/KeenClient 
Examples/objc/cocoapods/DerivedData/KeenClientExampleObjCCocoaPods/Build/Products/Debug-iphonesimulator/KeenClient/KeenClient.framework/KeenClient:
	@rpath/KeenClient.framework/KeenClient (compatibility version 1.0.0, current version 1.0.0)
	/System/Library/Frameworks/CoreLocation.framework/CoreLocation (compatibility version 1.0.0, current version 2101.0.62)
	/System/Library/Frameworks/Foundation.framework/Foundation (compatibility version 300.0.0, current version 1349.54.0)
	/usr/lib/libobjc.A.dylib (compatibility version 1.0.0, current version 228.0.0)
	/usr/lib/libSystem.dylib (compatibility version 1.0.0, current version 1238.50.2)
	/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation (compatibility version 150.0.0, current version 1349.55.0)
	/System/Library/Frameworks/SystemConfiguration.framework/SystemConfiguration (compatibility version 1.0.0, current version 888.50.20)
```
And here's CFNetwork in the loaded image list:
```
(lldb) image list
...
[ 65] 2A3090B3-6B46-3D7E-9873-A347640F5CFE 0x000000010f064000 /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk//System/Library/Frameworks/CFNetwork.framework/CFNetwork 
...
```